packages/@uppy/transloadit/src/index.ts

```diff
@@ -416,7 +416,7 @@ export default class Transloadit<
         .getFiles()
         .filter(({ id }) => fileIDs.includes(id))
 
-      if (files.length === 0) {
+      if (files.length !== fileIDs.length && files.length === 0) {
```

Comment:
Would it be clearer to write it like that?
```suggestion
      if (files.length === 0 && fileIDs.length !== 0) {
```

Reply:
Tested locally and this works too 👍